pkg/vpcmodel/externalNetwork.go

```diff
@@ -102,7 +104,6 @@ func newExternalNode(isPublicInternet bool, ipb *common.IPBlock) (Node, error) {
 	}
 	cidr := ipb.ToCidrList()[0]
 	return &ExternalNetwork{
-		VPCResource:      VPCResource{ResourceType: publicInternetNodeName},
```

Comment:
instead of deleting, set:  `resourceType: publicInternetNodeName`

Reply:
done